frontend/packages/webterminal-plugin/src/components/cloud-shell/WebTerminalConfiguration.tsx

```diff
@@ -0,0 +1,369 @@
+        ];
+      } else {
+        newEnvVariables = [{ name: 'WEB_TERMINAL_IDLE_TIMEOUT', value: timeout }];
+      }
+      terminalExecResource.container = {
+        ...terminalExecResource.container,
+        env: newEnvVariables,
+      };
+      return oldComponent;
+    }
+    if (terminalExecResource && terminalExecComponent?.value) {
+      terminalExecComponent.value = timeout;
+      return oldComponent;
+    }
+    return [
+      ...oldComponent,
+      {
+        name: 'web-terminal-exec',
+        container: {
+          env: [{ name: 'WEB_TERMINAL_IDLE_TIMEOUT', value: timeout }],
+          image: '',
+        },
+      },
+    ];
+  };
+
+  const submit = async () => {
+    setSaveStatus({ status: 'pending' });
+
+    const csTimeout = getCloudShellTimeout(value, unit) || '';
+    const newWebTerminalExec = {
+      ...webTerminalExec,
+      metadata: {
+        ...webTerminalExec?.metadata,
+        annotations: {
+          ...webTerminalExec?.metadata?.annotations,
+          'web-terminal.redhat.com/unmanaged-state': timeoutCheckBox ? 'true' : 'false',
+        },
+      },
+      spec: {
+        ...webTerminalExec?.spec,
+        components: getUpdatedComponentTimeout(webTerminalExec?.spec?.components, csTimeout),
+      },
+    };
+
+    const newWebTerminalTooling = {
+      ...webTerminalTooling,
+      metadata: {
+        ...webTerminalTooling?.metadata,
+        annotations: {
+          ...webTerminalTooling?.metadata?.annotations,
+          'web-terminal.redhat.com/unmanaged-state': imageCheckBox ? 'true' : 'false',
+        },
+      },
+      spec: {
+        ...webTerminalTooling?.spec,
+        components: getUpdatedComponentImage(webTerminalTooling?.spec?.components, image),
+      },
+    };
+    try {
+      k8sUpdateResource({
+        model: DevWorkspaceTemplateModel,
+        data: newWebTerminalExec,
+        name: 'web-terminal-exec',
+        ns: operatorNamespace || DEFAULT_NS_OPERATORS,
+      });
+      k8sUpdateResource({
+        model: DevWorkspaceTemplateModel,
+        data: newWebTerminalTooling,
+        name: 'web-terminal-tooling',
+        ns: operatorNamespace || DEFAULT_NS_OPERATORS,
+      });
+      fireTelemetryEvent('Console cluster configuration changed', {
+        customize: 'Web Terminal configuration items',
+        timeout: csTimeout,
+        image,
+      });
+      setSaveStatus({ status: 'successful' });
+    } catch (e) {
+      setSaveStatus({ status: 'error', error: e });
+    }
+  };
+  return (
+    <FormSection
+      title={t('webterminal-plugin~Web Terminal Configuration')}
+      data-test="web-terminal form-section"
+    >
+      <FormHelperText isHidden={false}>
+        {t(
+          'webterminal-plugin~As admin you can change the default timeout and image of Web Terminal.',
+        )}
+      </FormHelperText>
+      <FormGroup
+        label={t('webterminal-plugin~Timeout')}
+        helperText={t('webterminal-plugin~Set timeout for the terminal.')}
```

Comment:
I believe you can add prop `fieldId="timeout-value"` so that the field label is clickable and the number spinned is selected. Currently, nothing happen on click:

![Peek 2023-04-28 23-10](https://user-images.githubusercontent.com/139310/235254814-128ba987-5b95-48b2-b471-e1c43ffae5f1.gif)

Same for the Image